draft-packages/modal/KaizenDraft/Modal/Primitives/ModalFooter.scss

```diff
@@ -19,26 +15,26 @@
 
 .padded {
   padding: 0 $spacing-lg $spacing-lg $spacing-lg;
-  @media (max-width: $layout-breakpoints-medium) {
+  @media (max-width: $layout-breakpoints-medium - 1px) {
```

Comment:
Same comment here as in the other CSS file. You can flip these media queries from `max-width` to `min-width` to remove the `- 1px` - obviously flipping the CSS as well.